### PR TITLE
add unit test for layernorm, linear, outproductmean

### PR DESCRIPTION
**Pull request by oahzxl**

### CODE REVIEW

File: tests/test_fastnn/test_basic_ops.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
+import torch
+from torch import nn
+import math
+from typing import Optional, Callable
+import numpy as np
+from scipy.stats import truncnorm
+from fastfold.model.fastnn.ops import Linear as FastLinear
+
+
+class Linear(nn.Linear):
```
Comment (Gy-Lu) [inline tests/test_fastnn/test_basic_ops.py:10]: why not `from fastfold.model.nn import Linear`?

File: tests/test_fastnn/test_out_product_mean.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
+import torch
+from fastfold.model.fastnn.ops import OutProductMean as FastOutProductMean
+from torch import nn
+from typing import Optional
+from test_basic_ops import Linear
+import fastfold
+from fastfold.model.fastnn.ops import set_chunk_size
+
+
+class OuterProductMean(nn.Module):
```
Comment (Gy-Lu) [inline tests/test_fastnn/test_out_product_mean.py:10]: I think you can just import this class

**Comment by oahzxl** [comment]
截屏2022-10-12 11 40 57

File: tests/test_fastnn/test_out_product_mean.py (hunk 2 above)
Comment (oahzxl) [reply tests/test_fastnn/test_out_product_mean.py]: ok

**Review by Gy-Lu — APPROVED** [review]

**Pull request merged**